src/DroneSafetyApi/Services/IHeatmapFactory.cs

```diff
@@ -6,8 +6,17 @@
 
 namespace DroneSafetyApi.Services
 {
+    /// <summary>
+    /// The IHeatmapFactory interface is capable of generating new <see cref="IHeatmap"/>s
```

Comment:
I will mention the it is a factory design pattern, the more testable shouldn't go in this documentation